File: 2-semestre/uri-online-judge/c/1144.c
#include <stdio.h>
 
int main() {
    int n, i, quad = 1, cubo = 1;

    scanf("%d", &n);

    for (i = 1; i <= n; i++) {
        quad = i*i;
        cubo = i*i*i;
        printf("%d %d %d\n", i, quad, cubo);
        quad++;
        cubo++;
        printf("%d %d %d\n", i, quad, cubo);
    }
 
    return 0;
}

File: 2-semestre/uri-online-judge/javascript/1144.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const n = parseInt(lines.shift());
let quad;
let cubo;

for (let i = 1; i <= n; i++) {
    quad = Math.pow(i, 2);
    cubo = Math.pow(i, 3);

    console.log(`${i} ${quad} ${cubo}`);

    quad++;
    cubo++;
    
    console.log(`${i} ${quad} ${cubo}`);
}
